kubernetes/client/api_client.py

```diff
@@ -144,7 +144,8 @@ def __call_api(self, resource_path, method,
             body = self.sanitize_for_serialization(body)
 
         # request url
-        url = self.configuration.host + resource_path
+        # ignore trailing slashes in base url
+        url = self.configuration.host.rstrip('/') + resource_path
```

Comment:
I'm not sure if we should modify this file. It's generated by swagger-codegen (see the preamble of this file). @karlbunch suggested different solution (#388) - to fix the value when configuration object is created. How does it look?

Reply:
Haven't seen that notice. I thought about fixing this in configuration handling code too, but then rejected that idea. This approach isn't solving issue completely, since there is possibility that user will create configuration object manually (in python code).